### PR TITLE
PMM-6389 add VictoriaMetrics directory and empty config file

### PR DESCRIPTION
**Pull request by f41gh7**

### CODE REVIEW

File: ansible/playbook/tasks/update.yml [hunk 1]
```diff
@@ -124,6 +124,12 @@
     - name: Create working directory for Alertmanager
       file: path=/srv/alertmanager/data state=directory owner=pmm group=pmm
 
+    - name: Create working directory for VictoriaMetrics
+      file: path=/srv/victoriametrics/data state=directory owner=pmm group=pmm
+
+    - name: Create empty configuration file for VictoriaMetrics
+      file: path=/etc/victoriametrics-promscrape.yml state=touch owner=pmm group=pmm
```
Comment (tenmozes) [inline ansible/playbook/tasks/update.yml:131]: should this file have content `---`?

Comment (adivinho) [inline ansible/playbook/tasks/update.yml:131]: Could it be added checks that folder and file have not existed yet?

**Review by adivinho — CHANGES_REQUESTED** [review]

File: ansible/playbook/tasks/update.yml (hunk 1 above)
Comment (f41gh7) [reply ansible/playbook/tasks/update.yml:131]: I suppose, that file module already checks it and create directory only if its not exists. state=touch does the same.

**Review by adivinho — APPROVED** [review]

File: ansible/playbook/tasks/update.yml (hunk 1 above)
Comment (valyala) [reply ansible/playbook/tasks/update.yml:131]: It looks like it victoriametrics successfully reads empty yml file. @f41gh7 , could you confirm this?

**Pull request merged**